README.md

```diff
@@ -42,9 +42,7 @@ Note that the external-attacher does not scale with more replicas. Only one exte
 
 * `--leader-election`: Enables leader election. This is useful when there are multiple replicas of the same external-attacher running for one CSI driver. Only one of them may be active (=leader). A new leader will be re-elected when current leader dies or becomes unresponsive for ~15 seconds.
 
-* `--leader-election-namespace <namespace>`: Namespace where the external-attacher runs and where leader election object will be created. It is recommended that this parameter is populated from Kubernetes DownwardAPI.
-
-* `--leader-election-identity <id>`: Unique identity of this replica of external-attacher container. It must be unique among all running replicas of the external-attacher. Pod name populated from Kubernetes DownwardAPI is recommended.
```

Comment:
Move this to the Deprecated arguments section

Reply:
Ah sorry one more nit. This should be moved to the "Deprecated arguments" section

Reply:
yup, done